dev/deps/spark-deps-hadoop-2.7-hive-1.2

```diff
@@ -167,13 +168,13 @@ orc-shims-1.5.8.jar
 oro-2.0.8.jar
 osgi-resource-locator-1.0.3.jar
 paranamer-2.8.jar
-parquet-column-1.10.1.jar
-parquet-common-1.10.1.jar
-parquet-encoding-1.10.1.jar
-parquet-format-2.4.0.jar
-parquet-hadoop-1.10.1.jar
+parquet-column-1.11.0.jar
+parquet-common-1.11.0.jar
+parquet-encoding-1.11.0.jar
+parquet-format-structures-1.11.0.jar
+parquet-hadoop-1.11.0.jar
 parquet-hadoop-bundle-1.6.0.jar
```

Comment:
Is there a reason behind still using `1.6.0`? It means that parquet-mr `1.6.0` classes and `1.11.0` classes might be mixed in the classpath.

Reply:
Looks like we have to manage this version up too yes

Reply:
This is used by Hive serde tables when the built-in Hive is 1.2.1.